tests/unittest/test_stream.py

```diff
@@ -81,7 +86,7 @@ def test_prefetch_stream(num_prefetch, worker_type):
     path = os.path.join('tests', 'data', 'wikitext-2')
     token_path = os.path.join('tests', 'data', 'wikitext-2/*test*.tokens')
     test = nlp.data.WikiText2(segment='test', root=path)
-    corpus = nlp.data.CorpusStream(token_path, flatten=True, skip_empty=True, eos=EOS)
+    corpus = nlp.data.SimpleDatasetStream(nlp.data.CorpusDataset, token_path)
```

Comment:
missing flatten=True, skip_empty=True, eos=EOS?

Reply:
In this case we only want to test that prefetch works, so it doesn't matter which paramteres CorpusDataset is called with. Also, WikiText2 default changes from flatten=True to flatten=False due to the change of inheritance relationship (The lack of specifying flatten=True in the SimpleDatasetStream matches this).

It may be good to preserve the default flatten=True for WikiText2. If so, I'll also change this test case. @szha @eric-haibin-lin